css/all.css

```diff
@@ -0,0 +1,4556 @@
+/*!
+ * Font Awesome Free 5.13.0 by @fontawesome - https://fontawesome.com
+ * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+ */
+.fa,
```

Comment:
I noticed this css is at `fontawesome.css`. Maybe you forgot to remove it from this file.

Reply:
do I need to? It seems like it commented out. Will it be a problem?

Reply:
You need to, otherwise, the code will be loaded twice: from `all.css` and `fontawesome.css`